kb/additional-resources.md

```diff
@@ -1,16 +1,23 @@
 ---
-id: external-resources
-title: External Resources
-sidebar_label: External Resources
+slug: additional-resources
+title: Additional resources
+tags:
+  - resources
+date: 2022-12-12T00:00:00Z
 ---
 
-There are many ways to introduce and teach Temporal based on what background you are coming from. Temporal doesn't have a monopoly on explaining Temporal.
+<!-- truncate -->
```

Comment:
Move the truncate to below the first sentence so that the first sentence shows on the kb article landing page